### ISSUE TITLE
Update to Bevy 0.11

### ISSUE THREAD

**Issue by Carnagion** [MrGVSV/bevy_tileset_map#3]
Hi - I just came across this crate and `bevy_tileset`, and they seem amazing. Thank you for your work so far! Unfortunately, this one is quite outdated. Are there any plans to update it for the 0.11 version of Bevy?

**Comment by MrGVSV** [comment]
Unfortunately, I have a lot on my plate IRL right now so I don't think I'll have much time to do it myself anywhere in the near future :( 

I would definitely accept a PR to update either crate though!

**Comment by Carnagion** [comment]
Understandable - I'll try and see if I can update `bevy_tileset_map` somehow 😄

**Comment by MrGVSV** [comment]
Yeah sorry about that. If you are able to update it, feel free to make a PR!

Good luck!
